src/applications/financial-status-report/tests/unit/transform.unit.spec.jsx

```diff
@@ -0,0 +1,1106 @@
+        );
+      });
+      it('has valid data', () => {
+        const submissionObj = JSON.parse(transform(null, inputObject));
+        expect(submissionObj).haveOwnProperty('personalData');
+        expect(submissionObj.personalData).haveOwnProperty('employmentHistory');
+        expect(submissionObj.personalData.employmentHistory[0]).haveOwnProperty(
+          'veteranOrSpouse',
+        );
+        expect(
+          submissionObj.personalData.employmentHistory[0].veteranOrSpouse,
+        ).to.equal('VETERAN');
+        expect(
+          submissionObj.personalData.employmentHistory[0].occupationName,
+        ).to.equal('Full time');
+        expect(submissionObj.personalData.employmentHistory[0].from).to.equal(
+          '01/2017',
+        );
+        expect(submissionObj.personalData.employmentHistory[0].to).to.equal('');
+        expect(
+          submissionObj.personalData.employmentHistory[0].present,
+        ).to.equal(true);
+        expect(
+          submissionObj.personalData.employmentHistory[0].employerName,
+        ).to.equal('Veteran Current One');
+      });
+      describe('employer address', () => {
+        it('has valid structure', () => {
+          const submissionObj = JSON.parse(transform(null, inputObject));
+          expect(submissionObj).haveOwnProperty('personalData');
+          expect(submissionObj.personalData).haveOwnProperty(
+            'employmentHistory',
+          );
+          expect(
+            submissionObj.personalData.employmentHistory[0],
+          ).haveOwnProperty('employerAddress');
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress,
+          ).to.be.an('object');
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress,
+          ).haveOwnProperty('addresslineOne');
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress,
+          ).haveOwnProperty('addresslineTwo');
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress,
+          ).haveOwnProperty('addresslineThree');
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress,
+          ).haveOwnProperty('city');
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress,
+          ).haveOwnProperty('stateOrProvince');
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress,
+          ).haveOwnProperty('zipOrPostalCode');
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress,
+          ).haveOwnProperty('countryName');
+        });
+        it('has valid data', () => {
+          const submissionObj = JSON.parse(transform(null, inputObject));
+          expect(
+            submissionObj.personalData.employmentHistory[0].employerAddress
+              .addresslineOne,
+          ).to.equal('');
```

Comment:
Empty tests always look weird to me. Do we have a test in which these get populated?